README.md

```diff
@@ -2,11 +2,21 @@
 
 ## Getting Started
 
+### 1. Install Dependencies
 ```js
 $ npm install
-$ npm start
 ```
 
+### 2. Add Environment file
+
+You'll need to add an `environment.json` file to set your path to the gecko repository.
+
+```json
+{ "geckoPath" : "/Users/jlaster/src/mozilla/gecko-dev"}
```

Comment:
Hm, I posted here before but github lost my comment. Their service went down earlier today.

I don't think this config should one of the steps in the readme; I'd rather it be an optional footnote. So far we've gotten away with not needing any config, and it's one less step for someone to get up and running. What do you think of moving this to the bottom under a "Config" or "Advanced" section?